docs/source/changelog/0.6.0-changelog.rst

```diff
@@ -0,0 +1,439 @@
+******
+v0.6.0
+******
+
+:Date: May 01, 2021
+
+Contributors
+============
+
+A total of 42 people contributed to this
+release. People with a '+' by their names authored a patch for the first
+time.
+
+* Abel Aebker
+* Abhijith Muthyala
+* Adam Ryczkowski +
+* Alex Lembcke +
+* Anton Ballmaier +
```

Comment:
```suggestion
* Anton Ballmaier
```

I have authored a patch in 0.5 as well. Other names may be wrong as well?

Reply:
Can you link the PR, this may help to debug the dev_changelog.py script.

Reply:
Nvm, I found it: 
![image](https://user-images.githubusercontent.com/29168469/116787067-e2911380-aa56-11eb-997b-fd615f6e54c1.png)

The issue here is that your name changed between v0.4.0 -> v0.5.0 and v0.5.0 -> v0.6.0  from `AntonBallmaier` to `Anton Ballmaier` with a space -- our script works by pulling the author name and comparing author names from the set of past contributors to current contributors. I suppose we could do it based on a combination of email/user